_posts/2018-12-17-Flutter.md

```diff
@@ -0,0 +1,185 @@
+
+All the widgets are written in Dart.
+Because Flutter is an open source project, you can use the source code as a reference while applying the widgets.
+With Flutter you don’t have to worry anymore about support libraries to render your apps on old devices or about OEMs that decide to alter the platform widgets because they can.
+You can even enjoy the beauty of Material Design on devices that were released years before Material Design was introduced.
+Your app will also be future proof, as new design implementations of platform widgets won’t affect or break the layout of your app.
+<span class="image right"><img alt="JWorks widget" src="/img/2018-12-17-Flutter/jworks_widget.png"></span>
+If there are any breaking changes with future OS versions, then it’s a bug for Flutter to resolve instead for you.
+Flutter even added notch-support for the iPhone X before the phone was released.
+
+Here you can find a quick example how you can build up your layout by combining different widgets together.
+The JWorks widget is rendered inside a default Material app template on the iPhone XS.
+
+```dart
+import 'package:flutter/material.dart';
+
+class JWorksWidget extends StatelessWidget { // I'm a widget
+  @override
+  Widget build(BuildContext context) {
+    return Card( // I'm a widget too
+      elevation: 4.0,
+      child: Padding( // Yep, another widget
+        padding: const EdgeInsets.all(16.0),
+        child: Image.network( // We're all widgets!
+          'https://ordina-jworks.github.io/img/jworks/jworks-400x400.png',
+        ),
+      ),
+    );
+  }
+}
+```
+
+## One hybrid framework to rule them all
+
+On the 4th of December 2018 the first [Flutter Live](https://developers.google.com/events/flutter-live/){:target="_blank" rel="noopener noreferrer"} event was hosted.
+A lot of exciting announcements were made, which definitely shows that the Flutter team is determined to keep improving.
+[The first stable version was released](https://developers.googleblog.com/2018/12/flutter-10-googles-portable-ui-toolkit.html){:target="_blank" rel="noopener noreferrer"} almost two years after the first Alpha version was released.
+
+<img class="image fit" alt="Hummingbird" src="/img/2018-12-17-Flutter/hummingbird.png">
+
+Another huge announcement was revealed about the future plans of Flutter.
+The first step is [Hummingbird](https://medium.com/flutter-io/hummingbird-building-flutter-for-the-web-e687c2a023a8){:target="_blank" rel="noopener noreferrer"}, or Flutter for the web.
+Since Dart can also compile to JavaScript, this was a logical step to take.
+Being able to run on the web, you can also create a Progressive Web App with Flutter, so your mobile app which runs in the browsers becomes a web app.
+Appception right there.
+
+Google is also working to bring Flutter to desktop with Flutter Desktop Embedding.
+As of proof, the presentation of Flutter Live was running in a Flutter app on a laptop.
```

Comment:
"As of proof"?

Reply:
Sounds strange indeed, changed it to "To prove this concept"